cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -68,13 +67,18 @@
     <property>
         <name>twill.jvm.gc.opts</name>
         <value>-verbose:gc -Xloggc:&lt;LOG_DIR&gt;/gc.log -XX:+PrintGCDetails -XX:+PrintGCTimeStamps -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=1M</value>
-        <description>Java GC options for all Apache Twill containers</description>
+        <description>
+            Java garbage collection options for all Apache Twill containers; &lt;LOG_DIR&gt;
+            is the location of the log directory on each machine; note that the special
```

Comment:
Changing `is the location of the log directory on each machine` to `is the location of the log directory in the container` makes it clearer.